pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -979,19 +979,12 @@ func (p *Provider) CleanSSH(l *logger.Logger) error {
 	return nil
 }
 
-// ConfigSSH is part of the vm.Provider interface
+// ConfigSSH is part of the vm.Provider interface. For this provider,
+// it verifies that the test runner has a valid public SSH key, as
```

Comment:
Nit: "has a public SSH key", we don't actually know it's valid or even well-formed :)

Reply:
Fixed.